packages/map-template/src/components/LocationsList/LocationsList.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../variables.scss';
+
+.locations-list {
+
+    // A drag indicator as pseudo element
```

Comment:
Why this is needed here? It's already part of any bottom sheet.

It also makes a drag indicator appear on larger screens:

![image](https://github.com/MapsPeople/web-ui/assets/50314165/4add38db-171a-4833-9d78-0396e0590c18)

Reply:
It's needed due to the way the header is set to be sticky to the top of the list. The header has a white background and a z-index which covers the dragging indicator. 
However, it should not appear on desktop devices indeed. I will restrict it to only appear on mobile devices.

Reply:
Could we just make it have a transparent background instead of white?

Reply:
Hm, not really because it needs to have the white background so it does not overlap with the text underneath. 
![image](https://github.com/MapsPeople/web-ui/assets/113503166/29b69c58-76e9-4238-b7a4-f95afebb7a26)

Reply:
This is what we are aiming for.
![image](https://github.com/MapsPeople/web-ui/assets/113503166/615d0c57-fcc7-44f9-b489-7c9fd43cd275)

Reply:
Could we push the list itself down, so the header and list never overlaps? Maybe seek inspiration on the location details component.

Reply:
I have tried to use the grid to separate the header and the list so they don't overlap, but I did not have any success with that because of how this list is supposed to be presented.

Reply:
There was great success using a grid. Resolving this.